db_conn.py

```diff
@@ -3,12 +3,12 @@
 
 # ADD URI STRING HERE #################################
 # set production False, if you want to contribute to this repo and make changes
-production = True
+production = False
 
 uri_string =  ""
 
 #########################################################
 
 client = pymongo.MongoClient(uri_string)
-
+ 
```

Comment:
Remove this change. You have added a trailing whitespace.

Reply:
@vikasnitd remove this trailing whitespace.